images/dind-env-build-arm64.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -x
+# This code is porting from k8s.io/images/bootstrap/Dockerfile
+# Used to set Docker in docker environment
+
+#
+# BEGIN: DOCKER IN DOCKER SETUP
+#
+
+# Install Docker deps, some of these are already installed in the image but
+# that's fine since they won't re-install and we can reuse the code below
+# for another image someday.
+apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
@zhlhahaha I have to extract our builder image from kubevirt/kubevirt and make it multi-arch-fit. I think we can do something nice with `qemu-static-user`. Let me come back to you in a few hours.

Reply:
> @zhlhahaha I have to extract our builder image from kubevirt/kubevirt and make it multi-arch-fit. I think we can do something nice with `qemu-static-user`. Let me come back to you in a few hours.

Hi Roman, I understand your idea. But for prow utility images, do we need to build them for multi-arch?

Reply:
I would at least have the chance to build them for arm and ppc64 without having to care about the cluster-architecture. I am a little bit worried about  too much friction here, which may make it hard for us to update things in the future. Multi-arch image is basically just the little extra-plus which eases maintenance a little bit more.

Reply:
At least it looks to me like your spcripts expects this to be run on an actual arm machine.

Did you think about cross-compiling it with bazel? Something like

```
bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64_cgo ...
```

should already do the job.

Reply:
> I would at least want to have the chance to build them for arm and ppc64 without having to care about the cluster-architecture. I am a little bit worried about too much friction here, which may make it hard for us to update things in the future. Multi-arch image is basically just the little extra-plus which eases maintenance a little bit more.



> At least it looks to me like your spcripts expects this to be run on an actual arm machine.
> 
> Did you think about cross-compiling it with bazel? Something like
> 
> ```
> bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64_cgo ...
> ```
> 
> should already do the job.

Yes, as an arm server has joined in ci system, this script is using it to build arm image.
If we want to use "--platforms" to crossbuild prow utility **image** directly without any patch, it does not work, because the base image for prow utility image only support for x86. But I will try if we can use it crossbuild multiarch utility **binary**.